### PR TITLE
feat(release): Support Homebrew in goreleaser

### PR DESCRIPTION
**Pull request by rudeigerc**
Signed-off-by: Yuchen Cheng <rudeigerc@gmail.com>

Related to #347

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -89,3 +89,21 @@ docker_manifests:
...
+brews:
+  - name: envd
+    ids:
+    - envd
+    tap:
+      owner: tensorchord
+      name: homebrew-tap
+    commit_author:
+      name: Ce Gao
+      email: cegao@tensorchord.ai
```
Comment (gaocegege) [inline .goreleaser.yaml:101]: Is it possible to use org as the commit author here?

**Review by gaocegege — APPROVED** [review]
LGTM with nits

File: .goreleaser.yaml (hunk 1 above)
Comment (rudeigerc) [reply .goreleaser.yaml:101]: Sure. This section just sets the username and email of git in commit messages.

Comment (gaocegege) [reply .goreleaser.yaml:101]: Then let's use TensorChord and envd-maintainers@tensorchord.ai here

Comment (rudeigerc) [reply .goreleaser.yaml]: Got it.

**Comment by rudeigerc** [comment]
Closes #347

**Pull request merged**